src/commands/ui.rs

```diff
@@ -59,33 +88,39 @@ pub fn show_ui(options: &Options, args: &UI) -> anyhow::Result<()> {
     }
 }
 
+#[tokio::main]
+async fn open_url(url: &str) -> Result<reqwest::Response, reqwest::Error> {
+    reqwest::Client::new().get(url).send().await
```

Comment:
I would add `User-Agent: edgedb` like we use for downloads.
Also might use `HEAD` method instead, as you don't read the response body anyway.

Reply:
Sure thing! But unfortunately the server doesn't support `HEAD`.